src/components/MenuTab.tsx

```diff
@@ -0,0 +1,46 @@
+import clsx from "clsx";
+import type { ReactNode } from "react";
+import { useState } from "react"
+
+export type MenuItem = {
+  title: string
+  content: ReactNode
+  onChange?: (index: number) => void
+}
+
+export const MenuTab = ({ items, lightBackground, isSelected }: { items: Array<MenuItem>, lightBackground?: boolean, isSelected?: number }) => {
+  const [ currentIdx, setCurrentIdx ] = useState(0)
+  const equalSplitValue = 100 / items.length + "%"
+  const selectedIdx = isSelected ? isSelected : currentIdx
+  return (
+    <>
+      <div className="flex flex-row justify-center overflow-hidden">
+        { items.map((item, idx) => (
+          <>
+            <div
```

Comment:
It is a bad practive to use `div`'s for button purposes, prefer using "<button>". If you absolutely need to use a div for a button, remember to add `role="button"`, such that it is still tabbable.

Reply:
I prefer using `div`s here. I assume you would like to use links. I don't like preventing default for `a` with useless `href` attribute. However, I changed it to use links now. If you prefer `div`s with `role="button"`, then let me know :)